ota.c

```diff
@@ -39,6 +39,26 @@ void MyLoggingCallback(const int logLevel, const char* const logMessage) {
 bool userbeta=0;
 bool otabeta=0;
 
+char *strstr_lc(const char *full_string, const char *search) {
```

Comment:
please call this routine ota_strstr with a comment stating it is a lowercase version of strstr

Reply:
Ok.